src/genie/libs/parser/iosxe/tests/ShowWirelessFabricClientSummary/cli/equal/golden_output_expected.py

```diff
@@ -0,0 +1,269 @@
+expected_output = {
+    "number_of_fabric_clients": 33,
+    "clients": {
+        1: {
+            "mac_address": "58bf.ea71.80f0",
+            "ap_name": "a4-21-cap28",
+            "wlan": "19",
```

Comment:
if the value is always number, better to change to integer from string.

Reply:
Changed WLAN value to int type.

Reply:
confirmed